resolwe/test_helpers/test_runner.py

```diff
@@ -5,38 +5,298 @@
 ===================
 
 """
+import copy
+import logging
 import os
 import re
+import shutil
 import subprocess
-
+# Parallel django test execution is done with worker pools from the multiprocessing module;
+# since every worker process needs its own initialization code to start the manager infrastructure,
+# it also needs a teardown function, but the multiprocessing module fails to provide such
+# functionality publicly. Internally, the Finalize class imported above is used.
+from multiprocessing.util import Finalize  # undocumented
```

Comment:
I think using undocumented functionality is serious enough to warrant too many reminders rather than too little. People will likely skip the long comment and just stumble over the in-line 'undocumented' one, which will then prompt them to read the comment above. Alternatively, they'll read the long comment, misunderstand it, notice the undocumented in-line and re-read the long comment.